documentation/configure.markdown

```diff
@@ -274,6 +274,8 @@ Do not use the `username` or `password` settings.
 
 This controls how often, in minutes, to compact the database. The compaction process reclaims space and deletes unnecessary rows. If not supplied, the default is every 60 minutes.
 
+If `gc-interval` is set to 0, all GC processes will be disabled. This may be useful, for example, for a read-only PuppetDB instance using a replicated database. It should not be used for a PuppetDB instance that will receive any commands. 
```

Comment:
I don't think the part about receiving commands is accurate. The situation I'm familiar with is a single write database A shared by N PDB's dedicated to command processing, which each write to A and read off replicas of A. Under this scenario you may have a single PDB instance that does nothing but run GC on A, and receives no commands, but it's every instance _except_ that one on which you'd be setting gc-interval=0.